chore: internal docs for beta (#187)
storyblok/storyblok-cli: src/commands/languages/README.md

# Languages Command

> The `languages` command allows you to manage languages in your Storyblok space. Currently, it supports pulling language configurations from your space.

Reviewer: ```suggestion
The `languages` command allows you to handle internalization features within your Storyblok space. Currently, it supports pulling language configurations from your space.
```

Reviewer 2: Hey @jeremenichelli @alvarosabu, I think both versions are a bit too bold. As it doesn't allow for _managing_ anything at this point, we shouldn't raise any unrealistic expectations. If I'm not mistaken, this currently allows you to retrieve all configured languages as a JSON, similar to the `spaces/me` endpoint, correct?

Author: Correct, it only allows pulling the languages available on the space @manuelschroederdev

Reviewer 2: Are there any plans to expand the functionality in the future?

Author: @manuelschroederdev not in near future
